src/decision/decision_engine.h

```diff
@@ -142,23 +127,18 @@ class DecisionEngine {
   // External World helping us help the Strategies
 
   /**
-   * Add a list of assertions, as well as lemmas coming from preprocessing
-   * (ppLemmas) and pairwise the skolems they constrain (ppSkolems).
+   * Add that assertion is an (input) assertion, not corresponding to a
```

Comment:
This phrasing seems a bit odd. You mean in the sense of notifying, maybe change accordingly?

Reply:
BTW this pattern happens is several other places in the PR, could you change those as well?